### PR TITLE
MNT: Run PTH flake test in prep for supporting pathlib (visualization)

### PR DESCRIPTION
**Pull request by neutrinoceros**
### Description
Extracted from #16060. Ref #16924

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

File: astropy/visualization/scripts/fits2bitmap.py
Comment (pllim) [inline astropy/visualization/scripts/fits2bitmap.py:1]: Not sure what this script is for, so I'll let subpackage maintainer decide if supporting Path here needs a change log or not. Thanks!

Comment (astrofrog) [reply astropy/visualization/scripts/fits2bitmap.py:1]: I think we should probably just have a single changelog entry in 'other' which mentions that a number of functions now support Path, rather than fully enumerating all the functions that now support it.

**Review by astrofrog — APPROVED** [review]
Looks good to me though note the failure due to the missing changelog. I don't mind if we include one changelog entry per PR or if we mark them all as not needing one then add a single changelog entry later that says that many/all functions in astropy that accept filenames now accept paths.

**Comment by astrofrog** [comment]
Actually maybe worth just adding a changelog entry, it might not be that many functions in the end. Certainly it's not for this sub-package.

**Comment by neutrinoceros** [comment]
I'm confused: I didn't think the affected function was meant to be public ? I do see a `@deprecated_renamed_argument` decorator on it but it's never used directly and only documented as a command line script. Am I mistaken ?

**Comment by pllim** [comment]
Yeah I don't think we ship the scripts as "public API"? Merging. Thanks!

**Pull request merged**
